### PR TITLE
chore: Merge master branch into feature branch

### PR DESCRIPTION
**Pull request by dengfuping**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [x] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Merge master branch into feature branch.          |
| 🇨🇳 Chinese | 将 master 分支代码合并到 feature 分支。          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/config-provider/demo/size.md](https://github.com/ant-design/ant-design/blob/merge-master/components/config-provider/demo/size.md)
[View rendered components/config-provider/index.en-US.md](https://github.com/ant-design/ant-design/blob/merge-master/components/config-provider/index.en-US.md)
[View rendered components/config-provider/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/merge-master/components/config-provider/index.zh-CN.md)
[View rendered components/divider/demo/customize-style.md](https://github.com/ant-design/ant-design/blob/merge-master/components/divider/demo/customize-style.md)
[View rendered components/divider/demo/horizontal.md](https://github.com/ant-design/ant-design/blob/merge-master/components/divider/demo/horizontal.md)
[View rendered components/divider/demo/plain.md](https://github.com/ant-design/ant-design/blob/merge-master/components/divider/demo/plain.md)
[View rendered components/divider/demo/vertical.md](https://github.com/ant-design/ant-design/blob/merge-master/components/divider/demo/vertical.md)
[View rendered components/divider/demo/with-text.md](https://github.com/ant-design/ant-design/blob/merge-master/components/divider/demo/with-text.md)
[View rendered components/divider/index.en-US.md](https://github.com/ant-design/ant-design/blob/merge-master/components/divider/index.en-US.md)
[View rendered components/divider/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/merge-master/components/divider/index.zh-CN.md)
[View rendered components/form/index.en-US.md](https://github.com/ant-design/ant-design/blob/merge-master/components/form/index.en-US.md)
[View rendered components/form/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/merge-master/components/form/index.zh-CN.md)

### CODE REVIEW

**Comment by afc163** [comment]
分支错了，应该是合并到 feature 分支。

**Comment by afc163** [comment]
好像有 lint 错误。

File: components/table/interface.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
 import { CheckboxProps } from '../checkbox';
 import { PaginationConfig } from '../pagination';
 import { Breakpoint } from '../_util/responsiveObserve';
-
+import { INTERNAL_SELECTION_ITEM } from './hooks/useSelection';
```
Comment (afc163) [inline components/table/interface.tsx:10]: @dengfuping 这里有问题。

Comment (afc163) [reply components/table/interface.tsx:10]: image

**Comment by afc163** [comment]
merge-master 分支可以再合并一下 master，上面有新东西了。

File: components/table/interface.tsx (hunk 1 above)
Comment (dengfuping) [reply components/table/interface.tsx:10]: - CI 过了，求 approval

**Review by afc163 — APPROVED** [review]

**Pull request merged**